app/authenticated/help/template.hbs

```diff
@@ -48,7 +58,7 @@
 </section>
 
 <div class="over-hr">
-  <span><a href="http://rancher.com/learn/">Getting Started</a> with Rancher</span>
+  <span><a href="http://rancher.com/learn/" target="_blank">Getting Started</a> with Rancher</span>
```

Comment:
And all the ones in here too, at least the base URLs, like

company=http://rancher.com
github=https://github.com/rancher/rancher
docs=http://docs.rancher.com/rancher
forum=http://forums.rancher.com